source/release-notes/v1.8-release-notes.rst

```diff
@@ -102,38 +103,149 @@ The Open OnDemand Portal :ref:`ood-portal-generator-configuration` file now supp
+
+The data seen in extended view is extracted from the command ``qstat -j -r -xml``.
+
+Torque and Slurm schedulers now support QoS fields
 ...................................................
 
+Torque and Slurm schedulers now support the quality of service (QoS) fields.
+
+``qos`` is now an acceptable script attribute like so:
+
+.. code-block:: yaml
+
+   script:
+     qos: 'best'
+
+In Torque this translates to ``-l qos=best`` and in Slurm it is ``--qos best``.
+
 Shell hosts are now protected behind allowlists
 ...............................................
 
+Shell hosts are now protected behind allowlists. This means administrators now need
+to explicitly add hosts to an allowlist for users to be able to access them through
+the shell app.
+
+Any currently configured login host that is not hidden will be automatically allowed.
+However, administrators will need to add a configuration for compute nodes and any
+additional hosts they wish to allow access to.
+
+Administrators should also configure the default ssh host if they've not already done
+so because 'localhost' is no longer the out of the box default.
+
+:ref:`See the documentation for setting the default ssh host <set-default-ssh-host>`
+and :ref:`the documentation for setting the allowlist for more details <set-ssh-allowlist>`.
+
 Beta XDMOD integration in the dashboard
 .......................................
 
+If your site also uses `XDMOD`_ for job metric collection, Open OnDemand now
+has beta support to connect to that installation and show XDMOD panels in
+the dashboard.
+
+To enable this functionality simply set the ``OOD_XDMOD_HOST`` to it's full URL
+in ``/etc/ood/config/apps/dashboard/env``.
+
+.. code-block:: shell
+
+  OOD_XDMOD_HOST="https://xdmod.mysite.edu"
+
 Chinese localization
 ....................
 
+The Chinese localization ``zh-CN`` is now available in the dashboard and
+job composer applications for sites who wish to use it.
+
+To enable it, simply set the ``OOD_LOCALE`` environment variable globally
+in ``/etc/ood/config/nginx_stage.yml``
+
+.. code-block:: yaml
+
+  pun_custom_env:
+     OOD_LOCALE: "zh-CN"
+
+
 Ability to add additional info to batch connect app
 ...................................................
 
-Support for SVG logos
-.....................
+Batch connect app developers may now add files that add information the the batch
+connect panels.  :ref:`More information is documented here <bc_info_html_md_erb>`
+on batch connect view panels and this feature specifically.
+
+Support for SVG dashboard logos
+...............................
+
+Logos can now be SVG images and controlled through the ``OOD_DASHBOARD_LOGO_HEIGHT``
+configuration to resize them. :ref:`See documention on branding for form information. <branding>`
+
+Support for new Native VNC views
+................................
+
+Sites can now define instructions for connecting to VNC interactive jobs with native tools
+instead of using NoVNC through the browser.
```

Comment:
>Sites can now define instructions for connecting to VNC interactive jobs with native tools...

Sites can now provide directions for connecting to VNC interactive jobs using a native VNC client...